docs/Developers/Useful_Links.rst

```diff
@@ -0,0 +1,45 @@
+Useful Links
+============
+
+The purpose of this document is to gather links that coala
+developers usually use throughout their work.
+
+If you ever encounter a link that helped you or that is not
+a part of the document and should be, feel free to suggest it
+by creating an issue in our `issue tracker <https://github.com/coala/coala/issues/new>`_.
+
+Git-Links
+---------
+
+-   `Git Tutorial <https://try.github.io/levels/1/challenges/1>`_
+-   :doc:`coala's Git Tutorial<Git_Basics>`
+-   `Commit message guidelines <https://wiki.gnome.org/Git/CommitMessages>`_
+-   :doc:`coala Commits<Writing_Good_Commits>`
+-   `How to rebase <https://asciinema.org/a/78683>`_
+-   `Rebase Concept <http://jeffkreeftmeijer.com/2010/the-magical-and-not-harmful-rebase/>`_
+-   `Short Rebase Tutorial <http://stackoverflow.com/questions/7244321/how-do-i-update-a-github-forked-repository>`_
+-   `coala Git Repository <https://github.com/coala/>`_
+
+Python-Links
+------------
+
+-   `Code Style <https://www.python.org/dev/peps/pep-0008/>`_
+-   :doc:`coala Code Style<Codestyle>`
+-   `Python Tutorial <https://docs.python.org/3/tutorial/>`_
+-   `Shorter Tutorial Version <https://www.stavros.io/tutorials/python/>`_
+
+rST-Links
+---------
+
+-   `Basic rST <http://www.sphinx-doc.org/en/latest/rest.html>`_
+-   `Syntax <http://www.sphinx-doc.org/en/stable/markup/toctree.html#toctree-directive>`_
+
+coala-Links
+-----------
```

Comment:
I'd first and foremost include coala.io/links here which is an ever updating collection of the coala shortlinks!